package-lock.json

```diff
@@ -0,0 +1,11966 @@
+{
+  "name": "web3-plugin-eip4337",
+  "version": "0.1.0",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {
```

Comment:
There are both package-lock and yarn-lock files, you might be using npm and yarn both, I'll recommend to use only one package manager ( in this case yarn ).